### PR TITLE
toast component of mlan UI

### PR DESCRIPTION
**Pull request by devAmlan**
This is the toast component of the component library, Please review my PR.

### CODE REVIEW

File: library/component/toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    display:none;
+    justify-content:flex-start;
+    align-items:center;
+    padding-left:1rem;
+    position:fixed;
+    background-color:var(--primary-bgcolor);
+    color:var(--primary-color);
+    font-size:1.8rem;
+    border:1px solid var(--primary-color);
+    border-radius:5px;
```
Comment (saurabhan) [inline library/component/toast/toast.css:13]: use rem

**Review by saurabhan — APPROVED** [review]
use rem instead of px, use classes for styling and ids for js otherwise looks good

File: library/component/toast/toast.css (hunk 1 above)
Comment (devAmlan) [reply library/component/toast/toast.css:13]: Plz correct me if I'm wrong I heard somewhere that to use rem if it is >= 10px (1 rem). So can i use rem for 5 px?

Comment (saurabhan) [reply library/component/toast/toast.css:13]: tbh it's a personal choice, but rem enables resizing as per the user set font size so the UI does not mess up if the user has set font size anything other than what you had while developing.

**Review by developer-anand — APPROVED** [review]
Good to merge.

**Pull request merged**